frontend/src/components/shared/userPhoto/UserPhoto.tsx

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 import Avatar from '../../mocks/user.png';
 import './userPhoto.scss';
+import { UserPhotoInfo } from '../../interfaces/Interface';
 
-type UserPhotoProps = {
-  isOnline: boolean;
-  photoUrl: string;
-};
-
-export const UserPhoto: React.FC<UserPhotoProps> = props => {
+export const UserPhoto: React.FC<UserPhotoInfo> = ({ photoInfo }) => {
   return (
     <div className="user-photo">
       <img src={Avatar} alt="User avatar" />
-      {props.isOnline === true ? <span className="online-status"></span> : null}
+      {photoInfo.isOnline === true ? <span className="online-status"></span> : null}
```

Comment:
why do you write rewrote it? it was simple and clear

Reply:
photoInfo.isOnline === true really?